keel-slack/src/test/kotlin/com/netflix/spinnaker/keel/slack/SlackServiceTest.kt

```diff
@@ -1,80 +0,0 @@
-package com.netflix.spinnaker.keel.slack
-
-import com.netflix.spectator.api.NoopRegistry
-import com.netflix.spectator.api.Registry
-import com.netflix.spinnaker.config.SlackConfiguration
-import com.slack.api.model.block.LayoutBlock
-import com.slack.api.model.kotlin_extension.block.withBlocks
-import dev.minutest.junit.JUnit5Minutests
-import dev.minutest.rootContext
-import io.mockk.every
-import io.mockk.mockk
-import io.mockk.spyk
-import io.mockk.verify
-import org.springframework.core.env.Environment
-
-class SlackServiceTest: JUnit5Minutests {
```

Comment:
there's no value in testing slack services 🤷 . There's also no testing framework for it.